python/test/unit/language/test_core.py

```diff
@@ -3773,10 +3773,14 @@ def _kernel(dst, src, CACHE: tl.constexpr):
 
     if not is_cuda():
         if is_hip():
+            target_arch = get_arch()
+            # TODO: support gfx11 testing
+            if any(arch for arch in ["gfx11", "gfx12"] if arch in target_arch):
```

Comment:
Let's not worry about gfx11/12 archs here. For our internal CI, there is way it can skipped these tests if nessary.

Reply:
I enabled these tests only for gfx94. Is it Ok?